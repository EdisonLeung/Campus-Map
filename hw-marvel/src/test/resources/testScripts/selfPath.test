# Tests pathfinding where start node and end node are the same node

# Create Graph
CreateGraph selfPath

# Add nodes
AddNode selfPath a
AddNode selfPath b

# Add edge
AddEdge selfPath a b AB

# pathfinding from a to a
FindPath selfPath a a