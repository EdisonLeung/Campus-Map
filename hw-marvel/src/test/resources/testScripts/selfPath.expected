# Tests pathfinding where start node and end node are the same node

# Create Graph
created graph selfPath

# Add nodes
added node a to selfPath
added node b to selfPath

# Add edge
added edge AB from a to b in selfPath

# pathfinding from a to a
path from a to a: